python/ray/_private/gcs_utils.py

```diff
@@ -577,3 +577,24 @@ def use_gcs_for_bootstrap():
     This function is included for the purposes of backwards compatibility.
     """
     return True
+
+
+def cleanup_redis_storage(host, port, storage_namespace, **kwargs):
+    """This function is used to cleanup the storage. Before we having
+    a good design for storage backend, it can be used to delete the old
+    data. It support redis cluster and non cluster mode.
+
+    Args:
+       host: The host address of the Redis.
+       port: The port of the Redis.
+       storage_namespace: The namespace of the storage to be deleted.
+    """
+
+    import redis
+
+    try:
+        cli = redis.RedisCluster(host, port, **kwargs)
+    except redis.exceptions.RedisClusterException:
+        cli = redis.Redis(host, port, **kwargs)
+
+    cli.delete(storage_namespace)
```

Comment:
do we need handle failure and retry?

Reply:
> Quick question - Is the namespace in this PR the same as RAY_external_storage_namespace, that uses to add a prefix to the Redis keys? I am asking because some use patterns are using a shared Redis by a prefix.

It'll throw exception to the user, I think the user should just either retry or do something else.

Reply:
I assume the function won't raise if there's no data to delete?